src/frame/tests/index.test.ts

```diff
@@ -246,7 +248,8 @@ describe("frame index", () => {
                     "given name",
                     "list of user ids",
                     "list of group ids",
-                    true
+                    true,
+                    undefined
```

Comment:
Maybe add a test (new or one of the existing ones) where you actually pass a policy?

Reply:
ok.